crawler/containers_crawler.py

```diff
@@ -11,22 +11,25 @@ def __init__(self, feature_types, container):
         self.metadata['system_type'] = 'container'
 
 
-class ContainersCrawler(BaseCrawler):
+class ContainersCrawler(Crawler):
```

Comment:
BaseCrawler was definitely more helpful than generic Crawler, esp. to reflect inheritance and method overwriting.